java-client/src/main/java/org/opensearch/client/opensearch/cat/CatRequestBase.java

```diff
@@ -37,21 +37,60 @@
 package org.opensearch.client.opensearch.cat;
 
 import org.opensearch.client.opensearch._types.RequestBase;
+import org.opensearch.client.util.ObjectBuilder;
 import org.opensearch.client.util.ObjectBuilderBase;
 
-// typedef: cat._types.CatRequestBase
+import javax.annotation.Nullable;
+import java.util.HashMap;
+import java.util.Map;
 
+// typedef: cat._types.CatRequestBase
 
 
 public abstract class CatRequestBase extends RequestBase {
-	public CatRequestBase() {
-	}
 
-	protected abstract static class AbstractBuilder<BuilderT extends AbstractBuilder<BuilderT>>
-			extends
-				ObjectBuilderBase {
-		protected abstract BuilderT self();
+    @Nullable
+    private final String columnNames;
```

Comment:
Super minor, I think the APi refers to this property as `headers`, could we change it please?

Reply:
@reta are you sure that it will not ambiguity with request headers?

Reply:
I don't see request headers in `RequestBase` (afaik we do not use model classes to pass headers through), which request headers you are referring to?

Reply:
Perhaps the headers in question are the ones sent with the clients request? E.g "x-opaque-id" for example? @MarinaRazumovsky i am not sure if that's what you meant but I was going to make a similar comment. Correct me if i am wrong in my assumption.

Reply:
Thanks @AmiStrn , if that is the case, I think out of the context the `headers` could be misleading, but the context is `_cat` API [1] where such headers are related to the output table view. I am certainly fine with better name (if `headers` are confusing)  to align it with `_cat` APIs 

[1] https://opensearch.org/docs/2.6/api-reference/cat/index/

Reply:
makes sense, @MarinaRazumovsky wdyt? headers is the way it is described in the docs, and internal code as well.

Reply:
you are right, I rename this parameter